phylogenetic/rules/write_config.smk

```diff
@@ -0,0 +1,19 @@
+"""
+This part of the workflow writes run time configuration to a YAML file.
+
+OUTPUTS:
+
+    results/{build}/run_config.yaml
+"""
+
+rule write_config:
+    output:
+        config = "results/{build}/run_config.yaml",
```

Comment:
Hmm, wouldn't this be writing out the same config per `{build}`? Can it be run just once and just output as `results/run_config.yaml`?

Reply:
Good catch, removed `{build}` in [force-push](https://github.com/nextstrain/WNV/compare/3ffb4035ab96d45ad31be5fb772e4f3b527bd498..41271bb6d17a910b585638a58204b416159b0215)